api/dao/containerutil.py

```diff
@@ -133,6 +133,16 @@ def get_stats(cont, cont_type):
     return cont
 
 
+def get_analyses(acquisition_id, filename=None):
```

Comment:
Mind giving this a bit more of a descriptive name or description? And allow it to take container_name and container_id? I can see it being useful for more than just this example.

Reply:
Done.